util/helpers.py

```diff
@@ -7,3 +7,27 @@ def secs_to_hours(secs):
     hours, remainder = divmod(secs, 3600)
     minutes, seconds = divmod(remainder, 60)
     return '%d:%02d:%02d' % (hours, minutes, seconds)
+
+def check_ctcdecoder_version():
+    import sys # pylint: disable=import-outside-toplevel
```

Comment:
If you put the `# pylint` directive in its own line it'll apply to the entire function.